src/reference/asciidoc/mail.adoc

```diff
@@ -143,6 +143,10 @@ In this case, the only header populated is the mentioned above `IntegrationMessa
 Starting with version 5.5.11, the folder is closed automatically after `AbstractMailReceiver.receive()` if no messages received or all of them are filtered out independently of the `autoCloseFolder` flag.
 In this case there is nothing to produce downstream for possible logic around `IntegrationMessageHeaderAccessor.CLOSEABLE_RESOURCE` header.
 
+Starting with version 6.1, the `ImapIdleChannelAdapter` doesn't do an asynchronous message publishing anymore and respective `sendingTaskExecutor` option is deprecated for removal in the next version.
+This is necessary to block the idle listener loop for messages processing downstream (e.g. with big attachments) for which the mail folder must remain opened.
+If an async hand-off is appropriate, an `ExecutorChannel` as an output channel for this adapter is a recommended way from now on.
```

Comment:
This will need to be changed if you agree on the deprecation issue.

Reply:
Sorry, I do not. We will change this sentence in `6.2` respectively.